extensions/security/runtime/src/main/java/io/quarkus/security/runtime/interceptor/check/RolesAllowedCheck.java

```diff
@@ -54,7 +54,7 @@ public void apply(SecurityIdentity identity, Method method, Object[] parameters)
         Set<String> roles = identity.getRoles();
         if (roles != null) {
             for (String role : allowedRoles) {
-                if (roles.contains(role)) {
+                if (roles.contains(role) || ("**".equals(role) && !identity.isAnonymous())) {
```

Comment:
@Ladicek That looks fine, but do we also have to tweak `smallrye-jwt` code somewhere as you said it supports `*` ?

Reply:
Yea: https://github.com/smallrye/smallrye-jwt/blob/master/implementation/src/main/java/io/smallrye/jwt/auth/jaxrs/RolesAllowedFilter.java#L45

I'll submit a PR to SmallRye JWT once I verify that the SmallRye JWT code is actually used. The security stuff in Quarkus is hard to navigate :-)

Reply:
@Ladicek Cool, thanks for spotting it, that JAX-RS code is not used by `quarkus-smallrye-jwt`